Dockerfile

```
@@ -0,0 +1,6 @@
+FROM python:3.12-slim
+WORKDIR /auth-pack
+RUN pip install --no-cache-dir uv
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)